src/index.html

```diff
@@ -5,9 +5,54 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__photo">
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac 27 Retina"
+        >
+      </div>
+      
+      <div class="card__element">
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <p class="card__code">
+          Product code: 195434
+        </p>
+        <div class="card__rewievs">
+          <div class="stars">
+            <div class="stars stars--5">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <span class="card__rewievsTitle">
+            Reviews: 5
+          </span>
+        </div>
+        <div class="card__price">
+          <span class="card__word">
+            Price:
+          </span>
+          <span class="card__amound">
+            $2,199
+          </span>
+        </div>
```

Comment:
```suggestion
        <div class="card__price">
          <span class="card__word">
            Price:
          </span>
          
          <span class="card__amound">
            $2,199
          </span>
        </div>
```
3'rd point in checklist